client/layout/404.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
[According to our guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions), names should be `camelCase`, including abbreviations.
Also, if you search our code, `ReactDOM` is only used by you, while `ReactDom` is how it's usually imported. So even for consistency sake it's good to keep it camel case :)